packages/components/Menu/src/MenuList/MenuList.tsx

```diff
@@ -68,6 +71,10 @@ export const MenuList = compose<MenuListType>({
               </Slots.focusZone>
             </Slots.scrollView>
           </Slots.root>
+        ) : menuContext.hasMaxHeight ? (
```

Comment:
I'm contemplating if macOS should follow the same logic here.. @harrieshin @Saadnajmi @mischreiber

Reply:
opened a task for followup